src/api/app/controllers/webui/user_controller.rb

```diff
@@ -59,6 +59,7 @@ def show
     return unless User.current == @displayed_user
     @is_displayed_user = User.current == @displayed_user
     @patchinfos = @displayed_user.involved_patchinfos
+    @password_changable = !@configuration['change_password'] && !CONFIG['proxy_auth_mode'] && !CONFIG[:ldap_mode]
```

Comment:
> It works since it's not set to :off even if you set it to.

I don't understand what you mean?

Also ldap_mode is set to :off by default:

https://github.com/opensuse/open-build-service/blob/master/src/api/config/options.yml.example#L75